scripts/class_room.py

```diff
@@ -2,4 +2,37 @@
 
 
 class Room(Dojo):
-    pass
+    def __init__(self, room_type, room_name):
+        self.room_type = room_type
+        self.room_name = room_name
+        self.room_capacity = None
+        self.all_rooms = ['White', 'Blue', 'Violet', 'Gray']
+
+    def get_room_capacity(self):
+        if self.room_type.upper() != 'OFFICE':
+            self.room_capacity = 4
+        else:
+            self.room_capacity = 6
+        return self.room_capacity
+
+    def add_new_room(self):
+        if self.room_name not in self.all_rooms:
+            self.all_rooms.append(self.room_name)
+            if self.room_type.upper() != 'OFFICE':
+                return 'A {} called {} has been successfully created.'.format(
+                    self.room_type, self.room_name), self.all_rooms
+            else:
+                return 'An {} called {} has been successfully created.'.format(
+                    self.room_type, self.room_name), self.all_rooms
+        else:
+            return 'Room {} already exists.'.format(self.room_name), 0
+
+    def get_all_rooms(self):
```

Comment:
Oooh, I get it. Simulating the real world ++ Room isn't aware of other rooms, totally makes sense. ...cool, working on that right away. Thanks!